context/app/static/js/components/organ/Azimuth/Azimuth.jsx

```diff
@@ -32,7 +32,7 @@ function Azimuth({ config }) {
         <LabelledSectionText label="Nuclei in reference">{config.nunit}</LabelledSectionText>
         {/* eslint-disable react/no-danger */}
         <LabelledSectionText label="Reference dataset">
-          <span dangerouslySetInnerHTML={{ __html: dataRefHtml }} />
+          <StyledLink dangerouslySetInnerHTML={{ __html: dataRefHtml }} />
```

Comment:
Rather than introducing another single-use styled component, would `Box` + `sx` work here?

```suggestion
          <Box component="span" sx={(theme) => ({'&>a': { color: theme.palette.link.main })} dangerouslySetInnerHTML={{ __html: dataRefHtml }} />
```